apisix/plugins/limit-conn/init.lua

```diff
@@ -101,33 +98,46 @@ function _M.increase(conf, ctx)
     end
 end
 
-
 function _M.decrease(conf, ctx)
     local limit_conn = ctx.limit_conn
     if not limit_conn then
         return
     end
 
+    local is_http = ctx.config.subsystem == "http"
+    if not is_http then
+        core.log.warn("The limit-conn plugin is not applicable in stream mode")
+        return
+    end
```

Comment:
Why not put this on the top of the file, it could cache the result and no need to judge every time

Reply:
The modification is completed, thank you for your suggestion